src/rust/aggregator/sqlx_queries/order_history/init_last_indexed_timestamp.sql

```diff
@@ -0,0 +1,10 @@
+-- We can assume that everything before this timestamp was indexed, since
+-- there's nothing to index.
+INSERT INTO aggregator.order_history_last_indexed_timestamp ("time")
+SELECT
+    DATE_TRUNC('minute', "time")
+FROM
+    place_limit_order_events
```

Comment:
If there's no limit orders, this will return 0 rows, so nothing will be inserted, and no error would be raised. In the [Rust code](https://github.com/econia-labs/econia/pull/658/files/03a510d21ecbaff9dcc00fbf2e23f07779a2ce54#diff-e5ca5ff9be33fe28c5ce766144fe36c6af89372639b8002835f64e9be74e08e5R243-R256), this case is handled as well.